crates/vault-registry/src/lib.rs

```diff
@@ -577,8 +566,9 @@ impl<T: Config> Pallet<T> {
     }
 
     pub fn get_backing_collateral(vault_id: &T::AccountId) -> Result<Collateral<T>, DispatchError> {
-        let vault = Self::get_vault_from_id(vault_id)?;
-        Ok(vault.backing_collateral)
+        Ok(ext::staking::total_current_stake::<T>(vault_id)?
+            .try_into()
+            .map_err(|_| Error::<T>::TryIntoIntError)?)
```

Comment:
The vault's `backing_collateral` is now tracked via the staking pallet.

Reply:
I am not sure this is a good idea, given the discussions we had around SLA & multicollateral - we likely won't use collateral directly as stake. And when we change that, we'll have to re-introduce the `backing_collateral` field

Reply:
This is for the staking pool shared with nominators, where the stake is the collateral for issuing tokens.

Reply:
So basically, this would be `vault.collateral + sum(nominator.collateral)`? If nomination==flase then, it's just `vault.collateral`, right?

Should the staking module be called `collateral` instead? Since we are now using the term stake and collateral for one and the same thing.

Reply:
The staking pallet will also take the previous slashing / rewards as well. I would prefer to keep the naming since we also factor in the rewards and we may want to contribute this pallet to orml.

Reply:
with the pending changes to the spec, make sense!